crawl-ref/source/viewchar.cc

```diff
@@ -66,7 +66,11 @@ static const char32_t dchar_table[NUM_CSET][NUM_DCHAR_TYPES] =
         // arch .. item_food
         '\\',  '}',  '~',  '8',  '{',  '{',  '{',  '}',  ')',  '[',  '/',  '%',
         // item_scroll .. item_amulet
-         '?',  '=',  '!',  '(',  ':',  '|',  '|',  '}',  '%',  '%',  '$',  '"',
+         '?',  '=',  '!',  '(',  ':',  '|',
+#if TAG_MAJOR_VERSION == 34
```

Comment:
Huh, I didn't realize there were character sets where we didn't use `\` for rods. I suggest adding a comment to this explaining that it was for rods.